### PR TITLE
Change CI from Travis to Github actions

### PR DESCRIPTION
**Pull request by alex9smith**
https://trello.com/c/cnNEUjhz/2126-epic-migrate-travis-ci-usages-to-github-actions

### CODE REVIEW

**Comment by gidsg** [comment]
Can we remove coveralls dependency as that'll be no longer used?

**Comment by alex9smith** [comment]
> Can we remove coveralls dependency as that'll be no longer used?

Good catch!

File: requirements-dev.txt [hunk 1]
```diff
@@ -5,16 +5,11 @@
...
 #
 attrs==19.3.0             # via pytest, syrupy
-certifi==2020.6.20        # via -c requirements.txt, requests
-chardet==3.0.4            # via -c requirements.txt, requests
 click==7.1.2              # via -c requirements.txt, pip-tools
 colored==1.4.2            # via syrupy
-coverage==5.2.1           # via -r requirements-dev.in, coveralls, pytest-cov
-coveralls==2.1.1          # via -r requirements-dev.in
-docopt==0.6.2             # via -c requirements.txt, coveralls
+coverage==5.2.1           # via -r requirements-dev.in, pytest-cov
```
Comment (gidsg) [inline requirements-dev.txt:10]: Can we remove `pytest-cov`/`coverage`?

Comment (alex9smith) [reply requirements-dev.txt:10]: 🤦

**Review by gidsg — APPROVED** [review]
👍

**Pull request merged**